pkg/local_object_storage/writecache/flush_test.go

```diff
@@ -316,6 +318,27 @@ func newObject(t *testing.T, size int) (*object.Object, []byte) {
 	return obj, data
 }
 
+func initWC(t *testing.T, wc Cache) {
+	require.NoError(t, wc.Init())
+
+	obj, data := newObject(t, 1)
+	var err error
+
+	require.Eventually(t, func() bool {
+		var prm common.PutPrm
+		prm.Address = objectCore.AddressOf(obj)
+		prm.Object = obj
+		prm.RawData = data
+
+		_, err = wc.Put(prm)
+		return !errors.Is(err, ErrNotInitialized)
+	}, time.Second, 100*time.Millisecond)
```

Comment:
Is it enough with `-race` flag?

Reply:
it was not but because of another reason, fixed races